pybossa/auth/__init__.py

```diff
@@ -38,30 +39,45 @@
 assert blogpost
 assert auditlog
 
-class Requirement(object):
-    """ Checks a function call and raises an exception if the
-    function returns a non-True value. """
 
-    def __init__(self, wrapped):
-        self.wrapped = wrapped
+class Authorizer(object):
 
-    def __getattr__(self, attr):
-        real = getattr(self.wrapped, attr)
-        return Requirement(real)
+    actions = ['create', 'read', 'update', 'delete']
```

Comment:
Would it make more sense to use the following wording?
read -> get
create -> post
update -> put
delete -> del

What do you think?